source/functionality/update_gem.md

```diff
@@ -0,0 +1,72 @@
+# Update docs to use latest tech-docs-gem
```

Comment:
The only query I have is about this section: I think it's probably sufficient just to have the point about `bundle update` and not necessarily relate it to branches.  

Everything else in this PR is fine for current workflow for this project.

Reply:
agree